pkg/cmd/server/api/v1/types.go

```diff
@@ -331,6 +331,17 @@ type AuditConfig struct {
 	MaximumRetainedFiles int `json:"maximumRetainedFiles"`
 	// Maximum size in megabytes of the log file before it gets rotated. Defaults to 100MB.
 	MaximumFileSizeMegabytes int `json:"maximumFileSizeMegabytes"`
+
+	// Path to the file that defines the audit policy configuration.
+	PolicyFile string `json:"policyFile"`
```

Comment:
Don’t forget to add file fields to the list of external file references we have to resolve

Reply:
Hmm... where's that?

Reply:
Here https://github.com/openshift/origin/blob/master/pkg/cmd/server/api/helpers.go#L124